README.md

```diff
@@ -143,21 +196,22 @@ signed url. The diff below demonstrate how this is done based off of the previou
 examples:
 
 ```diff
-// RegistrationController.php
+// src/Security/EmailVerifier.php
 
-class RegistrationController extends AbstractController
+class EmailVerifier
 {
-    public function register(): Response
+    // ...
+
+    public function sendEmailConfirmation(string $verifyEmailRouteName, UserInterface $user, TemplatedEmail $email): void
     {
         $user = new User();
-    
+
         // handle the user registration form and persist the new user...
     
         $signatureComponents = $this->verifyEmailHelper->generateSignature(
```

Comment:
Don't we need to use that `$signatureComponents` anywhere? It seems we need to add `// ...` after this line?